roles/convert2rhel/tasks/activation_keys_sca_disabled.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: "Find subscriptions"
+  theforeman.foreman.subscription_info:
+    username: "{{ foreman_username }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    validate_certs: "{{ foreman_validate_certs | default(omit) }}"
+    organization: "{{ foreman_organization }}"
+    search: "name != {{ foreman_c2r_rhel7_product }} and name != {{ foreman_c2r_rhel8_product }} || "
+  register: subscriptions
+
+- name: "Create 'CentOS 7' activation key (SCA disabled)"
+  theforeman.foreman.activation_key:
+    username: "{{ foreman_username }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    validate_certs: "{{ foreman_validate_certs | default(omit) }}"
+    organization: "{{ foreman_organization }}"
+    name: "{{ foreman_c2r_ack_centos7 }}"
+    lifecycle_environment: "{{ foreman_c2r_lifecycle_env }}"
+    content_view: "{{ foreman_c2r_content_view }}"
+    auto_attach: false
+    subscriptions:
+      - name: "Convert2RHEL7"
+  when: foreman_c2r_enable_rhel7
+
+- name: "Create 'CentOS 8' activation key (SCA disabled)"
+  theforeman.foreman.activation_key:
+    username: "{{ foreman_username }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    validate_certs: "{{ foreman_validate_certs | default(omit) }}"
+    organization: "{{ foreman_organization }}"
+    name: "{{ foreman_c2r_ack_centos8 }}"
+    lifecycle_environment: "{{ foreman_c2r_lifecycle_env }}"
+    content_view: "{{ foreman_c2r_content_view }}"
+    auto_attach: false
+    subscriptions:
+      - name: "Convert2RHEL8"
+  when: foreman_c2r_enable_rhel8
+
+- name: "Create 'RHEL 7' activation key (SCA disabled)"
+  theforeman.foreman.activation_key:
+    username: "{{ foreman_username }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    validate_certs: "{{ foreman_validate_certs | default(omit) }}"
+    organization: "{{ foreman_organization }}"
+    name: "{{ foreman_c2r_ack_rhel7 }}"
+    lifecycle_environment: "{{ foreman_c2r_lifecycle_env }}"
+    content_view: "{{ foreman_c2r_content_view }}"
+    auto_attach: false
+    subscriptions: "{{ subscriptions['subscriptions'] | map('json_query','{ name: name } ') | list }}"
```

Comment:
`json_query` needs `jmespath`, and we can't depend on that.

I think `map(attribute='name')` should do the same?

Reply:
TBH I do not understand much how and why this Ansible stuff works like it works and this was the only way that it worked for me.
When I try `{{ subscriptions['subscriptions'] | map(attribute='name') | list }}` I'll get error:
`fatal: [localhost]: FAILED! => {"changed": false, "msg": "dictionary requested, could not parse JSON or key=value"}`